.github/workflows/php-cs-fixer.yml

```diff
@@ -0,0 +1,25 @@
+name: PHP-CS-Fixer
+
+on:
+    pull_request:
+    push:
+
+jobs:
+    php-cs-fixer:
+        runs-on: ubuntu-latest
+        steps:
+            -   name: Checkout
+                uses: actions/checkout@v2
+
+            -   name: Setup PHP
+                uses: shivammathur/setup-php@v2
+                with:
+                    php-version: 7.3
+                    coverage: none
+                    tools: php-cs-fixer:~2.16.0
+
+            -   name: Validate composer.json
+                run: composer validate
+
+            -   name: Run PHP-CS-Fixer
+                run: php-cs-fixer fix --dry-run --diff -v
```

Comment:
https://github.com/php-coveralls/php-coveralls/blob/master/.travis.yml#L39 is missing

Reply:
Nope - this is informed decision. 
Next step I think I will add phpmd run into `php-cs-fixer.yml` and will rename this file into something like `codestyle.yml`

Reply:
Applied `composer sca`